sdk/core/azure-core/tests/test_tracing_policy.py

```diff
@@ -40,7 +41,7 @@ def test_distributed_tracing_policy_solo(self, tracing_implementation, http_requ
             response.status_code = 202
             response.headers[policy._RESPONSE_ID] = "some request id"
 
-            assert request.headers.get("traceparent") == "123456789"
+            assert request.headers.get("traceparent") == "00-12345-GET-01"
```

Comment:
is this change in format just for the test or will it impact customers as well ( if they use it somehow)?

Reply:
The format remains the same. I just made the test traceparent value match the real format a bit more closely.